datastream/tools/verify_split.py

```diff
@@ -0,0 +1,14 @@
+import json
+from pathlib import Path
+
+
+def verify_split(old_path, new_path):
+    for old_split_name, old_split in json.loads(Path(old_path).read_text()).items():
+        for new_split_name, new_split in json.loads(Path(new_path).read_text()).items():
+            if old_split_name == new_split_name:
+                continue
+            if len(set(old_split).intersection(set(new_split))) > 0:
+                raise ValueError(
+                    f"Some keys from old split '{old_split_name}'"
```

Comment:
Might be nice to print the number of keys or even a list of the keys? (I think there's a nice function that wraps text that's too long, ask Jim)